### PR TITLE
Listimages

### PR DESCRIPTION
**Pull request by rjschwei**
Add a utility to list images and fix issues with the removeimg command and man page

### CODE REVIEW

File: ec2listimg [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    dest='version',
+    help='Program version'
+)
+
+args = argparse.parse_args()
+
+if args.version:
+    version_file_name = 'VERSION'
+    base_path = os.path.dirname(utils.__file__)
+    version = open(base_path + os.sep + version_file_name, 'r').read()
```
Comment (smarlowucf) [inline ec2listimg:117]: Nitpick: it's nice to strip the newline before output otherwise the output ends up like 'version'\n\n.

File: ec2listimg [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    print(version)
+    sys.exit(0)
+
+config_file = os.path.expanduser(args.config_file_path)
+config = None
+if not os.path.isfile(config_file):
+    print('Configuration file "%s" not found.' % config_file)
+    sys.exit(1)
+try:
```
Comment (smarlowucf) [inline ec2listimg:126]: A space would be nice before "try:" for readability.

File: ec2listimg [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    access_key=access_key,
+    image_id=args.image_id,
+    image_name=args.image_name,
+    image_name_fragment=args.image_name_frag,
+    image_name_match=args.image_name_match,
+    secret_key=secret_key,
+    verbose=args.verbose)
+
+if len(regions) > 1:
+    lister.set_intent(4)
```
Comment (smarlowucf) [inline ec2listimg:179]: s/set_intent/set_indent

File: lib/ec2imgutils/ec2listimg.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+            if self.verbose == 0:
+                print(output + image.get('Name'))
+            elif self.verbose == 1:
+                print(output + image.get('Name') + '\t' + image.get('ImageId'))
+            else:
+                pp = pprint.PrettyPrinter(indent=(4 + self.indent))
+                pp.pprint(image)
+
+    # ---------------------------------------------------------------------
+    def set_intent(self, indent):
```
Comment (smarlowucf) [inline lib/ec2imgutils/ec2listimg.py:91]: s/set_intent/set_indent

File: lib/ec2imgutils/ec2listimg.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+import pprint
+
+import ec2imgutils.ec2utils as utils
+from ec2imgutils.ec2imgutils import EC2ImgUtils
+from ec2imgutils.ec2imgutilsExceptions import EC2ListImgException
+
+
+class EC2ListImage(EC2ImgUtils):
+    """List image in account."""
```
Comment (smarlowucf) [inline lib/ec2imgutils/ec2listimg.py:26]: s/image/images

File: lib/ec2imgutils/ec2listimg.py [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+            except Exception:
+                msg = 'Unable to complie regular expression "%s"'
+                msg = msg % self.image_name_match
+                raise EC2ListImgException(msg)
+        else:
+            return owned_images
+
+    # ---------------------------------------------------------------------
+    def list_images(self):
+        """List th eimages that match in the accoount"""
```
Comment (smarlowucf) [inline lib/ec2imgutils/ec2listimg.py:76]: s/th eimages/the images
s/accoount/account

File: lib/ec2imgutils/ec2listimg.py [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+                    self.image_name_match)
+            except Exception:
+                msg = 'Unable to complie regular expression "%s"'
+                msg = msg % self.image_name_match
+                raise EC2ListImgException(msg)
+        else:
+            return owned_images
+
+    # ---------------------------------------------------------------------
+    def list_images(self):
```
Comment (smarlowucf) [inline lib/ec2imgutils/ec2listimg.py:75]: I missed this initially. It's probably better for this info to be returned from method. The Binary would print out output but if calling from API we probably want `images` returned from this method.

**Comment by rjschwei** [comment]
Needs some rework to consider #41. Since #41 is a lot more work it should go first and then this needs to be merged and reworked to use the logger. Also the simple verbose=0|1|2 idea will no longer work with the logger in place so that has to be re-thought. And the spec file is missing the inclusion of the new util.

**Comment by rjschwei** [comment]
Should be ready to go.

**Comment by smarlowucf** [comment]
$ flake8

./lib/ec2imgutils/ec2listimg.py:105:1: W293 blank line contains whitespace
ec2listimg:21:1: F401 'logging' imported but unused

**Comment by rjschwei** [comment]
Fixed

**Pull request merged**